doc/source/data/performance-tips.rst

```diff
@@ -99,7 +99,7 @@ just two of the five columns of Iris dataset.
 Parquet Row Pruning
 ~~~~~~~~~~~~~~~~~~~
 
-Similarly, you can pass in a filter to ``ray.data.read_parquet()`` (selection pushdown)
+Similarly, you can pass in a filter to ``ray.data.read_parquet()`` (filter pushdown)
```

Comment:
selection pushdown is confusing in data world, as it normally means projection to me. Other systems (such as Spark, Presto, Parquet, etc) are always using filter pushdown. We are also using filter pushdown in other places e.g. [here](https://docs.ray.io/en/master/data/creating-datasets.html#dataset-deferred-reading)

Reply:
Agreed here!